proteomic_aggregation/python/proteomic_aggregation.ipynb

```diff
@@ -0,0 +1,3591 @@
+{
```

Comment:
<p>Suggestion for 3rd sentence:&nbsp;<em>mass spectrometry based </em><strong><em>proteomic </em></strong><em>analyses</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>